packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -2,6 +2,64 @@
+
+#### Domains API is now GA in Preview
+
+The [Domains API](/docs/reference/api/domains/) is now GA in Preview. This API allows you to customize your Okta org domain name for a seamless branded experience so that your users see the same base URL for your application. <!--OKTA-386752-->
+
+#### Domains API supports OAuth 2.0
+
+The [Domains API](/docs/reference/api/domains/) has been updated to support OAuth 2.0. You can grant access to the Domains API using the `okta.domains.manage` and the `okta.domains.read` scopes instead of using SSWS tokens. <!--OKTA-381286-->
+
+#### Okta Sign-In Widget, version 5.6.0
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.6.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-391046-->
+
+#### Refresh Token Rotation is now GA in Preview
+
+[Refresh Token Rotation](/docs/guides/refresh-tokens/refresh-token-rotation/) is now GA in Preview. Refresh Token Rotation helps a public client to securely rotate refresh tokens after each use. When refresh token rotation behavior is enabled in Okta, a new refresh token is returned each time the client makes a request to exchange a refresh token for a new access token.
+<!--OKTA-390933-->
+
+#### SAML parameter SessionNotOnOrAfter now available
+
+The SAML parameter `SessionNotOnOrAfter` is now GA for SAML assertions and is available for use with Okta’s [SAML Inline Hooks](/docs/reference/saml-hook/). This optional parameter specifies the session lifetime, in seconds, and is included in the SAML assertion. The `SessionNotOnOrAfter` parameter allows the Identity Provider to control the session of the Service Provider. Most SAML applications manage their own sessions. However, some SAML applications require this parameter from the Identity Provider for session management. <!--OKTA-390950-->
+
+#### System Log API SCIM filter expression update
+
+The [System Log API](/docs/reference/api/system-log/) has been updated to return an HTTP 400 response code when a request is made with a SCIM filter expression using the `co` (contains) operator with the `debugContext.debugData.url` or `debugContext.debugData.requestUri` field values. Use an alternative filter expression field such as `actor.id` or `target.id` to filter events in the System Log API. <!--OKTA-388434-->
```

Comment:
has been updated > is updated

Reply:
present perfect sounds a bit better here?

Reply:
Yes, I guess it's OK.